recirq/engine_utils.py

```diff
@@ -377,42 +377,3 @@ async def worker():
 
 def _get_current_time():
     return datetime.datetime.now()
-
-
-def get_available_processors(processor_names: List[str]):
-    """Returns a list of available processors.
-
-    Checks the reservation status of the processors and returns a list of
-    processors that are available to run on at the present time.
-
-    Args:
-        processor_names: A list of processor names which are keys from QUANTUM_PROCESSORS.
-    """
-    project_id = os.environ['GOOGLE_CLOUD_PROJECT']
-    engine = cg.get_engine()
-    available_processors = []
-    current_time = _get_current_time()
-    for processor_name in processor_names:
-        processor_id = get_processor_id_by_device_name(processor_name)
-        if processor_id is None:
-            # Skip the check if this is a simulator.
-            continue
-        processor = engine.get_processor(processor_id)
-        for time_slot in processor.get_schedule():
-            try:
-                time_slot = cg.EngineTimeSlot.from_proto(time_slot)
```

Comment:
It looks like nothing is using this function, so I guess it's okay to delete it.  I think the goal was to have a quantum chess interface that could opportunistically use open swim slots to run moves when the processor was available, but I doubt that will happen now.

If you want to keep this function, you can just get rid of `cg.EngineTimeSlot.from_proto(time_slot)` and operate directly on the `QuantumTimeSlot` which should be very similar.  I think if you just delete this line, it might work (or be very close to working).  If you want to take this opportunity to prune the dead code though, that's also fine with me.

Reply:
I think `get_schedule` has the wrong return type annotation. why doesn't mypy pick it up?

Reply:
I got 

```
E               TypeError: '<' not supported between instances of 'Timestamp' and 'datetime.datetime'
```

I tried changing 
```
def _get_current_time():
    from google.protobuf.timestamp_pb2 import Timestamp
    timestamp = Timestamp()
    return timestamp.GetCurrentTime()
```

which confusingly did not change the error. I think I'm going to give up and continue with the removal.